src/content/tools/dart-compile.md

```diff
@@ -164,6 +169,54 @@ Run your compiled app from the `/tmp` directory:
 $ ./tmp/myapp
 ```
 
+#### Cross-compilation {: #cross-compilation-exe }
+
+Cross-compilation to Linux x64 and ARM64 is supported on the
+following 64-bit host operating systems: macOS, Windows,
+and Linux.
+
+To use cross-compilation, include the following flags:
+
+*   `--target-os=linux`: The target operating system
+    for the compiled executable. The Linux operating system
+    is supported at this time.
+
+*   `--target-arch=value`: The target
+    architecture for the compiled executable. The value
+    for this flag can be `arm64` (64-bit ARM processor)
+    or `x64` (64-bit processor).
+
+*   `--experimental-cross-compilation`: Explicitly enables
```

Comment:
Could please remove `--experimental-cross-compilation` mentions here and everywhere? We've decided to remove the flag

Reply:
Absolutely. Done!